examples/lessons/1/index.js

```diff
@@ -2,21 +2,13 @@
 
 window.webGLStart = function() {
 
-  // ES5 replacement of
-  //  import {Program, Camera, createGLContext} = PhiloGL;
-  var Program = PhiloGL.Program;
-  var Camera = PhiloGL.Camera;
-  var createGLContext = PhiloGL.createGLContext;
+  var pgl = PhiloGL;
```

Comment:
A major purpose of this refactor is to phase out the Philocreate a number of collaborative ES6 classes that can be independently imported and then freely composed with each other, or with other gl libraries. 

Because of this, I'd prefer to drop the prefix/namespace in the code, and introduce a variable for each class/function as originally suggested.

In the end, there will probably still be a namespace, but it may be called something different than PhiloGL, so having a pgl variable will be a little misleading.